### ISSUE TITLE
Cleanup docs build configuration for old versions

### ISSUE THREAD

**Issue by charith-elastic** [elastic/cloud-on-k8s#3448]
We should look into the implications of using the `7.x.asciidoc` include file for 0.8 and 0.9 (https://github.com/elastic/docs/pull/1893#pullrequestreview-449905727). Off the top of my head, I don't think it matters because we did not start using those attributes in docs until after 0.9 (#1747) but we should make sure.

At the same time, we should decide whether we want to keep the 0.8 and 0.9 documentation available on the website as we have now released several versions that supersede them.

**Comment by anyasabo** [comment]
I am +1 to removing 0.8 and 0.9 docs

**Comment by gtback** [comment]
Typically we try to leave all historical versions of docs (we still have Elasticsearch documentation back to 0.90). It only adds minimal overhead to the build process, and old versions are marked with a banner indicating that newer docs are available. Looking at analytics data, we still get some sporadic hits on ECK 0.9.

**Comment by alaudazzi** [comment]
If the content from 0.8 and 0.9 no longer applies, I would remove these versions. They might only generate confusion.

**Comment by pebrc** [comment]
The content still applies nothing has changed in that regard. But we don't want users to actually use these versions anymore but there are still users on these versions which is why I believe we should not remove the docs.

**Comment by charith-elastic** [comment]
I removed the include for `7.x.asciidoc` and ran the docs build for both 0.8 and 0.9. There were no errors so I am reasonably confident that the include statement is superfluous. 

This means that we can completely remove the entry for `7.x.asciidoc` from the build configuration because we don't require any of our documentation to be rebuilt when that file changes.

One of the reasons why I proposed removing the 0.8 and 0.9 versions was because I assumed they were adding this extra configuration complexity when trying to add a new version to the docs. Since that doesn't seem to be the case anymore, I am now in favour of keeping them online.
